tests/integration_tests/performance/conftest.py

```diff
@@ -16,23 +16,7 @@
 
 
 # pylint: disable=too-few-public-methods
-class ResultsDumperInterface:
```

Comment:
Thanks!

Reply:
haha, yeah I wanted to tie the results_dir thing in with the results dumper first, and half-way through noticed that the perf tests don't actually put things into sub directories. I'd already made this commit by then though lol